src/configs/base_keyspecs.c4m

```diff
@@ -631,6 +631,50 @@ time of chalk mark insertion.
 """
 }
 
+keyspec AUTHOR {
+    kind:             ChalkTimeArtifact
+    type:             string
+    standard:         true
+    since:            "0.1.2"
+    shortdoc:         "Author of most recent commit"
+    doc:              """
+The author of the most recent commit in the current branch, which is likely the same as the committer but not always.
+"""
+}
+
+keyspec AUTHOR_DATE {
```

Comment:
Should we call it `DATE_AUTHORED` instead?  AUTHOR_DATE feels like my birthday :)

Reply:
Happy to change these, would better keys be:
`DATE_AUTHORED` and `DATE_COMMITTED` ?

Reply:
Yeah those sound much clearer.